app/com/linkedin/drelephant/schedulers/AirflowScheduler.java

```diff
@@ -58,10 +62,14 @@ public AirflowScheduler(String appId, Properties properties, SchedulerConfigurat
   }
 
   private void loadInfo(String appId, Properties properties) {
-    // Update the 4 Ids
+    // examples:
+    // my_amazing_task_id
     _taskId = properties.getProperty(AIRFLOW_TASK_ID);
+    // 2016-06-27T01:30:00
```

Comment:
Just kidding! I was wrong. So `${_dagId}` and `${_taskId}` are at most 250 char each so we are fine. Since the dates are only 19 characters the longest any Id can ever be is 250*2+19*2+3=541 (for a jobExecId with max length dag/task ids).